dataland-frontend/src/components/forms/parts/UploadReports.vue

```diff
@@ -147,42 +160,60 @@ export default defineComponent({
       }
     },
     /**
-     * Handles selection of a file by the user. If duplicates are found in the selection, this is handled.
+     * Handles selection of files by the user. If invalid file names are found in the selection, this is handled.
```

Comment:
Duplicates are also handled.

Reply:
I have extended this js doc to make clear that an invalid file name can have two reasons (forbidden character or duplicate name)

Reply:
Perfect. I guess I misunderstood the meaning of invalid file names first.